.github/workflows/app.yml

```diff
@@ -0,0 +1,36 @@
+name: Test and build Jellyfin Android
+on:
+  push:
+  release:
+    types: [ published ]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Cache Gradle wrapper and dependencies
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/wrapper/dists
+            ~/.gradle/caches
+          key: gradle-cache
+      - name: Set environment variable with version
+        if: ${{ startsWith(github.ref, 'refs/tags/v') }}
+        run: |
+          echo 'JELLYFIN_VERSION<<EOF' >> $GITHUB_ENV
+          echo $GITHUB_REF | sed 's|refs/tags/||' >> $GITHUB_ENV
+          echo 'EOF' >> $GITHUB_ENV
+      - name: Test and assemble APKs
+        run: ./gradlew test assemble versionTxt
+      - name: Publish compiled APKs as artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: jellyfin-apks
+          path: app/build/outputs/apk/**/jellyfin-android-*.apk
+      - name: Publish version.txt as artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: version.txt
+          path: app/build/version.txt
```

Comment:
I'd rather create 1 artifact with the apk files and version.txt together. Basically the "complete package" to upload to repo.jellyfin.org (like https://repo.jellyfin.org/releases/client/android/versions/v2.2.2/)

Reply:
That should be possible, I just took the easiest approach here.

Reply:
Done.